app/helpers/link_helper.rb

```diff
@@ -14,10 +14,10 @@ def link_to_next_question(questionnaire)
   def link_to_next_module_item(item, link_args = { class: 'govuk-button' })
     mod = item.training_module
 
-    if item.next_item
-      link_to item.next_button_text, training_module_content_page_path(mod, item.next_item), link_args
+    if item.next_item.certificate?
```

Comment:
Because the named path is the same for both now this conditional should move to `next_button_text`

Reply:
I don't follow

Reply:
ok, assuming I guessed correct, I've simplified the link_to_next_module_item and now just use the same next_button_text (which now checks for certificate)